data/human/human missions.txt

```diff
@@ -6891,3 +6891,370 @@ mission "Returning to Paradise"
 			action
 				payment 120000
 			`You're not sure if you've ever met a more pretentious and entitled family. The girls fought non-stop, but fortunately, the family was satisfied with the luxury accommodations on your ship. Once you land on <planet>, the lady gives you <payment> and the family leave without a word of thanks; they never even tell you their names. Nonetheless, you're happy to be rid of them.`
+
+
+
+mission "Adelita 1"
+	name "Exhibition installations to <planet>"
+	description `Transport <tons> of exhibition installations to <destination> for the Adelita collective. Payment is <payment>.`
+	minor
+	cargo "exhibition installations" 5
+	source "Follower"
+	destination "Moonshake"
+	to offer
+		or
+			random < 10
+			and
+				random < 60
+				has "Expedition to Hope 2: done"
+		"combat rating" > 40
+
+	on offer
+		conversation
+			branch "has visited"
+				has "visited planet: Hope"
+			`Walking on the edges of <origin>'s spaceport district, you notice signage advertising an exhibition about the planet Hope and the disaster and evacuation that happened on it in 2994; you vaguely remember it as being a planet in a system southwest of New Boston. But what makes it stand out from the typical advertisements that you see on notice boards and interactive screens is some of its colorful language: "Considered Offensive by Paradisians," "Why We Can't Give Up Hope," and "Evicted by the Paradise Planets."`
```

Comment:
I think its explained in FW what happened (by Katya iirc?) and this mission doesnt seem to check for that

Reply:
Yeah, I could update it to add another branch if you've done Katya's missions but still haven't been to Hope.